src/apsp_johnson/src/apsp_johnson.c

```diff
@@ -254,7 +267,7 @@ int compute_apsp_johnson(char* sql, edge_apsp_johnson_t **output_edges,
 }
 
 PG_FUNCTION_INFO_V1(apsp_johnson);
-Datum apsp_johnson(PG_FUNCTION_ARGS) {
+PGDLLEXPORT Datum apsp_johnson(PG_FUNCTION_ARGS) {
```

Comment:
I think that `PGDLLEXPORT` is defined in PostgreSQL itself,
so above modification will become duplicate.
https://github.com/postgres/postgres/search?utf8=%E2%9C%93&q=PGDLLEXPORT